### PR TITLE
Add documentation for avatar creation/upload endpoints

### PR DESCRIPTION
**Pull request by th3fallen**
Refs: https://github.com/wheniwork/wheniwork-app/pull/2975

### CODE REVIEW

File: source/methods/login.md.erb [hunk 1]
```diff
@@ -0,0 +1,63 @@
+## Create/Update User Avatar
```
Comment (shadowhand) [inline source/methods/login.md.erb]: This should say "Login Avatar" I think.

File: source/methods/login.md.erb [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    },
+    "created_at": "2016-01-13T22:05:22+00:00",
+    "updated_at": "2016-01-13T22:05:22+00:00",
+    "best": "medium"
+  },
+  "error": false
+}
+<% end %>
+
+Creates or updates user avatar.
```
Comment (shadowhand) [inline source/methods/login.md.erb]: "login"

File: source/methods/login.md.erb [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+> Example Response
+
+<% json do %>
+{
+  "avatar": {
+    "hash": "{hash}",
+    "type": "jpg",
+    "sizes": {
+      "small": {
+        "href": "{userhash}/small",
```
Comment (shadowhand) [inline source/methods/login.md.erb]: `loginhash`

File: source/methods/login.md.erb [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+        "path": "2/fb/{userhash}.small.jpg",
```
Comment (shadowhand) [inline source/methods/login.md.erb]: etc

**Comment by th3fallen** [comment]
@shadowhand thanks for catching that.

**Pull request merged**